NetTally.Core/Output/TallyOutput.cs

```diff
@@ -69,6 +69,11 @@ private void BuildGlobal(CancellationToken token)
 
             using (new Spoiler(sb, "Tally Results", DisplayMode == DisplayMode.SpoilerAll || AdvancedOptions.Instance.GlobalSpoilers))
             {
+                // Formating options
+                // If output is constructed, further output is skipped
+                if (ConstructFormattingOutput(token))

```

Comment:
Move this to after AddHeader, since the header provides additional important info (which posts were tallied, any active filters, which version of the program is used, etc).